data/pigui/modules/system-view-ui.lua

```diff
@@ -667,21 +651,9 @@ function Windows.objectInfo.Show()
 end
 
 function Windows.objectInfo.Dummy()
-	ui.text(lc.OBJECT_INFO)
-	ui.spacing()
-	ui.separator()
-	ui.spacing()
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
-	ui.text("TAB LINE")
+	ui.withFont(detailfont, function()
+		ui.text("Tiny rocky planet with no significant")
+	end)
```

Comment:
If possible, I'd recommend increasing the vertical height of the Object Info window - at 1440x900 I'm consistently getting a scrollbar when selecting planets / moons even with small or no descriptions. Nothing else uses the vertical space on that side of the screen as far as I'm aware.

Reply:
oh, it turns out I put the bracket in the wrong place, this window should have an adaptive height, will fix.